### PR TITLE
Integration Tests 2

### PR DESCRIPTION
**Pull request by MihanixA**

### CODE REVIEW

File: contracts/vaults/UniV3Vault.sol [hunk 1]
```diff
@@ -225,27 +225,6 @@ contract UniV3Vault is IUniV3Vault, IntegrationVault {
         actualTokenAmounts[1] = amounts.a1;
     }
 
-    function _getLiquidityForAmounts(
```
Comment (alexeuler) [inline contracts/vaults/UniV3Vault.sol:228]: how's the best effort pull fixed if this is deleted?

**Review by alexeuler — CHANGES_REQUESTED** [review]

File: contracts/vaults/UniV3Vault.sol (hunk 1 above)
Comment (MihanixA) [reply contracts/vaults/UniV3Vault.sol:228]: I think it is not a problem when UniV3Vault goes last.

**Comment by MihanixA** [comment]
[Yearn, Aave prop tests](https://trello.com/c/g1c3ruJq/308-yearn-aave-prop-tests)

**Comment by MihanixA** [comment]
[UniV3 best effort pull](https://trello.com/c/kpm4MFj5/304-univ3-best-effort-pull)

**Review by alexeuler — APPROVED** [review]

**Pull request merged**